run.sh

```diff
@@ -1,11 +1,11 @@
-module load ReFrame/3.10.1
+module load ReFrame/4.1.0
```

Comment:
there is already a module for ReFrame 4.2.0, can we use that?

Reply:
Why not